phys2bids/interfaces/txt.py

```diff
@@ -76,7 +76,7 @@ def process_labchart(channel_list, chtrig, header=[]):
     channel_list: list
         list with channels only
     chtrig : int
-        index of trigger channel
+        index of trigger channel, starting in 1 for human readability
```

Comment:
And at this point, should we state that their count starts in 1, but zero is a correct input - and what it does?
(Here and in every other docstring)